.travis.yml

```diff
@@ -0,0 +1,9 @@
+
+language: go
+go: 
+ - tip
```

Comment:
I'd have the opposite concern, things might work with a newer version, while not working in a released version, the one we'll officially use to build releases of the tool.

I've seen this happen with 1.4 -> 1.5, 1.6 transition.

E.g. one can build code with `tip` that references new fields that don't exist in the latest release.

OTOH Go has the Go1 compatibility clause that guarantees existing code will keep building with newer versions in the 1.x release family.